pages/docs/learn/durable-execution.mdx

```diff
@@ -0,0 +1,34 @@
+export const description = 'Learn how Durable Execution works with Inngest. '
+
+# How Durable Execution works with Inngest
+
+One of the core features of Inngest is Durable Execution. Durable Execution allows your functions to be fault-tolerant and resilient to failures. The end result is that your code, and therefore, your overall application, is more reliable.
+
+In this guide, we'll cover the basics of Durable Execution, how it works, and how it works with Inngest functions.
+
+{/* Note - this page is written a specific way for search optimization */}
+## What is Durable Execution?
+
+Durable Execution is a fault-tolerant approach to executing code that handles failures and interruptions gracefully through automatic retries and state persistence. This means that your code can continue to run even if there are issues like network failures, timeouts, infrastructure outages, and other transient errors.
```

Comment:
I'd split the first sentence into two here. Short sentences are preferable when explaining new concepts. (thinking of an alternative that also doesn't involve redundant duplication or a passive voice 🤪 )

Reply:
Yeah - I reworked this a lot, but I was thinking that the approach (the how) is important to what it is. Instead of "through" it could be "achieved with" which may be easier to read 🤔 /